ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -243,7 +247,7 @@ - (void)validatedAcquireToken:(ADAuthenticationCallback)completionBlock
         return;
     }
     
-    if (![ADAuthenticationContext isForcedAuthorization:_promptBehavior] && !_skipCache && [_context hasCacheStore])
```

Comment:
I'm wondering if there was a scenario where cacheStore is not available?

Reply:
I didn't find such scenario, the cache was always initialized.